datafusion/physical-expr/src/window/cume_dist.rs

```diff
@@ -34,11 +34,15 @@ use std::sync::Arc;
 #[derive(Debug)]
 pub struct CumeDist {
     name: String,
+    data_type: DataType,
```

Comment:
I think it would help to note that this is the output type, not the argument type

```suggestion
    /// Output data type
    data_type: DataType,
```

Reply:
will do for all functions